src/Microsoft.SqlTools.ServiceLayer/ObjectExplorer/SmoModel/TreeNodeGenerator.tt

```diff
@@ -179,6 +179,12 @@ namespace Microsoft.SqlTools.ServiceLayer.ObjectExplorer.SmoModel
 				{
 					WriteLine("            var child = new SmoTreeNode();");
 					WriteLine("            child.IsAlwaysLeaf = true;");
+					if(type.EndsWith("s")) 
```

Comment:
this logic is for the node types that don't have specific class for creating node, Folders, tables, views they have specific class but column, triggers ...  don't have it

---
In reply to: [112316790](https://github.com/Microsoft/sqltoolsservice/pull/323#discussion_r112316790) [](ancestors = 112316790)